scapy/layers/dot11.py

```diff
@@ -174,7 +174,7 @@ def mysummary(self):
     def guess_payload_class(self, payload):
         if self.type == 0x02 and (0x08 <= self.subtype <= 0xF and self.subtype != 0xD):
             return Dot11QoS
-	elif self.FCfield & 0x40:
+        elif self.FCfield & 0x40:
```

Comment:
Hi, please do not alter the lines you don't actually modify, since this breaks the code history.